pymatgen/io/openmm/parameterizer.py

```diff
@@ -0,0 +1,441 @@
+            parameterizer_assignment: ParameterizerAssignment = ParameterizerAssignment.INFERRED,
+            customize_force_field: bool = False,
+            custom_file_paths: Optional[List[str]] = None,
+            **kwargs
+            ):
+            """
+            Initialize a Parameterizer instance for parameterizing a molecular system.
+
+            Parameters
+            ----------
+            topology : tk.Topology
+                The molecular topology.
+            mol_specs : list
+                A list of molecule specifications.
+            box : list
+                A list of the box dimensions.
+            force_fields : list of str
+                A list of the force field names.
+            parameterizer_types : dict of str: ParameterizerType, optional
+                A dictionary mapping force field names to their corresponding parameterizer type. If not specified,
+                all force fields will be assigned the default parameterizer type.
+            parameterizer_assignment : ParameterizerAssignment, optional
+                The method used to assign parameterizer types to the force fields. Options are "INFERRED", "EXPLICIT",
+                and "DEFAULT". Defaults to "INFERRED".
+            customize_force_field : bool, optional
+                Whether or not to use custom force field files. Defaults to False.
+            custom_file_paths : list of str, optional
+                A list of custom force field file paths. Only used if customize_force_field is True.
+            **kwargs
+                Additional keyword arguments to pass to the OpenMM `ForceField.createSystem` method, if applicable.
+
+            Raises
+            ------
+            AssertionError
+                If the parameterizer type is not of the correct type.
+                If multiple parameterizer types are being used for the force fields.
+                If an explicit parameterizer type is required but not provided.
+                If the force field type is not supported by the parameterizer.
+                If the topology type is unexpected for the parameterizer type.
+                If custom force field files are being used with a non-OpenMM parameterizer.
+                If no custom force field files are provided when using custom force fields.
+
+            Notes
+            -----
+            The `mol_specs` argument should be a list of dictionaries, where each dictionary contains the following
+            Examples
+            --------
+            >>> parameterizer = Parameterizer(
+                    topology,
+                    mol_specs=[
+                        {
+                            "smile": "O",
+                            "force_field": "amoeba",
+                            "count": 200
+                        },
+                        {
+                            "smile":"[Cl-]",
+                            "force_field": "amoeba",
```

Comment:
What's the behavior if no force_field is specified?

It'd be great to have an option to specify a default FF for all mol_specs